docs/filters.md

```diff
@@ -222,6 +224,18 @@ Matches issues which were created on a given date, or within a given date range.
 
 Matches issues which were updated in the given number of hours. For example, `updated:<24` would match issues updated in the last day. If a number `n` is given, it is implicitly translated to `<n`.
 
+### updatedByOthers
+
+*Expects a number or  number range*
+
+Matches issues which were updated in the given number of hours by users other than the logged in user. For example, `updated:<24` would match issues updated in the last day by other users. If a number `n` is given, it is implicitly translated to `<n`.
+
```

Comment:
What's meant by 'number range'? Something like `2-4`? In that case, give an example.